src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,35 @@
+.nav {
+  &__list {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 32px;
+  }
+
+  &__link {
+    color: inherit;
```

Comment:
Consider adding 'font-size' and 'font-weight' properties to style the navigation links.